packager/media/formats/packed_audio/packed_audio_writer_unittest.cc

```diff
@@ -42,8 +42,10 @@ const uint32_t kTimescale = 9000;
 const char kOutputFile[] = "memory://test.aac";
 // For multi-segment mode.
 const char kSegmentTemplate[] = "memory://test_$Number$.aac";
-const char kSegment1Name[] = "memory://test_1.aac";
-const char kSegment2Name[] = "memory://test_2.aac";
+const char kSegment124Name[] = "memory://test_124.aac";
+const char kSegment125Name[] = "memory://test_125.aac";
+const int64_t kSegmentIndex123 = 123;
```

Comment:
Can we start the segment index from 0 or 1?

Reply:
Can do that. Do you want to pass segment number to MediaHandlerTestBase::GetSegmentInfo ?

Reply:
Ah, didn't notice that. It is fine to keep it as is then.